src/hacbs/consts/pipelinerun.ts

```diff
@@ -15,6 +15,12 @@ export enum PipelineRunLabel {
   COMMIT_PROVIDER_LABEL = 'pipelinesascode.tekton.dev/git-provider',
   COMMIT_SHA_TITLE_ANNOTATION = 'pipelinesascode.tekton.dev/sha-title',
   COMMIT_TYPE_LABEL = 'pipelines.appstudio.openshift.io/type',
+
+  TEST_SERVICE_APPLICATION = 'test.appstudio.openshift.io/application', // TODO: should be removed once the backend removes the prefix - https://issues.redhat.com/browse/HACBS-1331.
+  TEST_SERVICE_COMPONENT = 'test.appstudio.openshift.io/component', // TODO: should be removed once the backend removes the prefix - https://issues.redhat.com/browse/HACBS-1331.
```

Comment:
This should be tracked in a jira.

Reply:
@karthikjeeyar Is this being tracked in Jira?

Reply:
https://issues.redhat.com/browse/HACBS-1393, blocked this ticket with backend ticket.